src/operator/tensor/elemwise_binary_scalar_op_extended.cc

```diff
@@ -49,8 +49,47 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_minimum_scalar)
     .set_attr_parser(ParamParser<NumpyBinaryScalarParam>)
     .set_attr<FCompute>("FCompute<cpu>", BinaryScalarOp::Backward<cpu, mshadow_op::le>);
 
+#if MXNET_USE_ONEDNN == 1
+bool PowerStorageType(const nnvm::NodeAttrs& attrs,
+                      const int dev_mask,
+                      DispatchMode* dispatch_mode,
+                      std::vector<int>* inputs,
+                      std::vector<int>* outputs) {
+  CHECK_EQ(inputs->size(), 1);
+  CHECK_EQ(outputs->size(), 1);
+
+  return DNNLStorageType(attrs, dev_mask, true, dispatch_mode, inputs, outputs);
+}
+
+void DNNLPowerForward(const nnvm::NodeAttrs& attrs,
```

Comment:
Why is this function declared here? I believe that usually we add the declaration of this kind of functions to dnnl_ops-inl.h file and definition to dnnl_*op_name*.cc file. Is this operator's implementation somehow preventing that or maybe is there any other reason to do it in this way?

Reply:
Definition and implementation are moved to dnnl files now.